config/locales/en.yml

```diff
@@ -73,6 +73,17 @@ en:
           label: "Yes, I’ve had problems with drug addiction or alcohol addiction (alcoholism)"
         option_no:
           label: "No"
+    know_nhs_number:
+      title: "Do you know your NHS number?"
+      hint: |
+        An NHS number is a 10 digit number, like 485 777 3456.
+        <br /><br />
```

Comment:
This is rather atypical as we don't tend to have hints with multiple paragraphs. I think this should do for now.

Reply:
Thanks Alex!